src/CMakeLists.txt

```diff
@@ -329,6 +329,7 @@ if( MIOPEN_BACKEND MATCHES "OpenCL" OR MIOPEN_BACKEND STREQUAL "HIPOC" OR MIOPEN
         ${COMPOSABLE_KERNEL_SOURCE}
         ${COMPOSABLE_KERNEL_DYNAMIC_ASM_SOURCE}
         ${COMPOSABLE_KERNEL_DYNAMIC_CPP_SOURCE}
+        kernels/detect_llvm_amdgcn_buffer_atomic_fadd_f32_float.cpp
```

Comment:
@asroy I've moved the file here because the detection feature is not composable-kernel-specific.

Reply:
That's good.